projects/addon-table/directives/table-filters/table-filters.directive.ts

```diff
@@ -0,0 +1,40 @@
+import {Directive} from '@angular/core';
+import {identity, merge, Observable, ReplaySubject} from 'rxjs';
+import {map, startWith, switchMap} from 'rxjs/operators';
+
+import {TuiTableFilterDirective} from './table-filter.directive';
```

Comment:
I am not sure but it can produce problem in Angular 11.

You import from `table-filter.directive.ts` here. And inside `table-filter.directive.ts` you import from `table-filters.directive.ts`.
Possible problem: https://angular.io/errors/NG3003

Could you please locally merge your branch with `v3.x`-branch and check it ?

P.S. There is chance that no error will produce (because you import entities only for typings).

Reply:
I'll just add interface :)